docs/faq/lukso/general-information.md

```diff
@@ -0,0 +1,42 @@
+
+### Why did LUKSO launch as Layer-1 Blockchain?
+
+LUKSO is an L1 blockchain designed to provide **distinct security, interoperability, and adaptability advantages**, creating a network effect that supports a newly independent economy without compromising its key aspects.
+
+LUKSO is dedicated to constructing standards that **significantly enhance the user experience of blockchain** technology. However, the full potential of Universal Profiles can't be realized if only a fraction of the network is onboarded through it. Interactions between profiles bring substantial value, and making Universal Profiles the initial accounts through which users are joining guarantees easy usability and upgradability for everyone. Creating a new network offers a space to **evolve and refine** these new concepts **without the hindrance of constraints** from current landscapes and without the obligation of being fully backward compatible.
+
+LUKSO also wants to ensure **smooth user onboarding** and subsidize early adopters using Universal Profiles. Creating a network effect of convenience through subsidization would not have been sufficient on existing Layer-1 blockchains due to high gas costs or Universal Profiles being a smaller subset of deployed accounts.
+
+Despite L2 solutions offering lower costs, they often suffer from interoperability issues, governance concerns, and the need for rolling back data to the main network, **risking the siloing of assets and profiles**. Depending on the rollup type, different sacrifices like calculation, smart contract execution, or finality time have to be made on top.
+
+Rollups, while providing scalability, present their challenges. They are only as secure as the speed at which assets can be exited to the main chain. Moreover, **not every protocol or activity** implemented on the rollup **can be exited**, as not everything is an asset. This limits their applicability and functionality for new account systems.
+
+While standalone applications might not see the necessity of having their own Layer-1 network, LUKSO provides a **distinct realm for a new** core blockchain **account ecosystem**, necessarily needing the robust and secure foundation to utilize its full potential.
+
+### What's the token distribution of LUKSO?
+
+Please look at our official [Network Migration](https://etherscan.io/address/0x42000421dd80D1e90E56E87e6eE18D7770b9F8cC#code) article to see the distribution amounts and listing of the final supply. Previous distributions have been outlaid in our [Genesis Validator](https://medium.com/lukso/its-happening-the-genesis-validators-are-coming-ce5e07935df6) announcement.
```

Comment:
is it the correct link? I would expect a link to an article, maybe you wanted to put this?

https://medium.com/lukso/the-lyxe-migration-process-374053e5ddf5